sigma/plugins/utility/other/define.py

```diff
@@ -35,11 +35,14 @@
 
     try:    
         if '<span class="intro-colon">' not in str(soupObj):
-            raise IndexError("The website has changed, please fix.")
+            raise IndexError('The website has changed, please fix. ("intro-colon")')
 
         for tag in soupObj.find_all(class_="intro-colon"):
             definition += tag.text
             if(len(tag.next_sibling.strip()) == 0):
+                if '<span class="sx-link sc">' not in str(soupObj):
+                    raise IndexError('The website has changed, please fix. ("sx-link sc")')
```

Comment:
That wouldn't work, because some definitions don't have hyperlinked definitions.